packages/core/content-releases/server/src/services/release.ts

```diff
@@ -48,7 +48,11 @@ const createReleaseService = ({ strapi }: { strapi: LoadedStrapi }) => ({
     const updatedRelease = await setCreatorFields({ user, isEdition: true })(releaseData);
 
     const release = await strapi.entityService.update(RELEASE_MODEL_UID, id, {
-      // @ts-expect-error Type 'ReleaseUpdateArgs' has no properties in common with type 'Partial<Input<"plugin::content-releases.release">>'
+      /*
+       * The type returned from the enttiy service: Partial<Input<"plugin::content-releases.release">>
+       * is not compatible with the type the type we are passing here: UpdateRelease.Request['body']
+       */
+      // @ts-expect-error see above
```

Comment:
Sorry to be annoying here. But why do we have this situation? What's the root cause, and why can't we find a proper TS fix?

also you have a typo, `enttiy`

Reply:
I'm not sure. The entity service doesn't seem to be aware of the attributes on a Release. It has them for create though... @Feranchz might be able to answer better.

Reply:
It's exactly that, as I understand this `Partial<Input<"plugin::content-releases.release">>` should be basically an interface with all the fields from release optionals... But for some reason right now it's actually returning a type with only the id (The only thing you can't update)